src/views/GameTemplates/GameTemplates.scss

```diff
@@ -0,0 +1,75 @@
+@import "../../index";
+
+.template-header {
+    display: flex;
+    color: $color-white;
+    font-size: 2rem;
+    justify-content: space-evenly;
+
+    .input-field {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        padding: .5em;
+
+        input {
+            height: 50px;
+            width: 300px;
+            font-size: 1rem;
+            border: round;
+            border-color: white;
+            // border-radius: 10px;
+            box-shadow: 6px 6px 14px -5px rgba(0, 0, 0, 0.5);
+            background-color: transparent;
+            color: white;
+        }
+    }
+}
+
```

Comment:
Using the new `div` in the component, we can add a new rule for putting margins on the page:
```
.page-container {
    margin-left: 40px;
    margin-right: 40px;
}
```